src/scales/linearScale.ts

```diff
@@ -25,6 +25,14 @@ namespace Plottable.Scales {
       return this._d3Scale(value);
     }
 
+    public scaleTransformation(value: number) {
+      return this.scale(value);
+    }
+
+    public getTransformationDomain() {
+      return this.domain() as [number, number];
```

Comment:
Scale's `this.domain` returns `D[]` where `D` is the templated domain type. It should probably return [D, D], but I don't want to redo all the other instances.